pkg/asset/installconfig/ibmcloud/client.go

```diff
@@ -377,6 +397,34 @@ func (c *Client) GetVPC(ctx context.Context, vpcID string) (*vpcv1.VPC, error) {
 	return nil, &VPCResourceNotFoundError{}
 }
 
+// GetVPCs gets all VPCs
+func (c *Client) GetVPCs(ctx context.Context) ([]vpcv1.VPC, error) {
+	_, cancel := context.WithTimeout(ctx, 1*time.Minute)
+	defer cancel()
+
+	regions, err := c.getVPCRegions(ctx)
+	if err != nil {
+		return nil, err
+	}
+
+	allVPCs := []vpcv1.VPC{}
+	for _, region := range regions {
+		err := c.setVPCServiceURLForRegion(ctx, *region.Name)
```

Comment:
I will need to rebase this PR in order to fix this call now that the necessary PR has merged,
https://github.com/openshift/installer/pull/6032
```
pkg/asset/installconfig/ibmcloud/client.go:413:11: c.setVPCServiceURLForRegion undefined (type *Client has no field or method setVPCServiceURLForRegion, but does have SetVPCServiceURLForRegion)
```